File: week_1/04_is_number_exist.py
input = [3, 5, 6, 1, 2, 4]


def is_number_exist(number, array):
    for n in array: #array 길이만큼 연산
        if n==number: #1번 연산
            return True
    return False

result = is_number_exist(9, input)
print(result)

File: week_1/04_is_number_exist.cpp
#include <iostream>
using namespace std;

int arr[6] = { 3, 5, 6, 1, 2, 4 };

bool is_number_exist(int number, int* arr) {
	bool check=false;
	for (int i = 0; i < 6; i++) { //하나씩 비교
		if (arr[i] == number)
			return true;
	}
	return false;
}

int main(void) {
	cout << is_number_exist(9, arr) << endl;
	return 0;
}
